pubsub/mqtt/mqtt.go

```diff
@@ -88,6 +88,7 @@ func (m *mqttPubSub) Init(metadata pubsub.Metadata) error {
 	}
 
 	m.ctx, m.cancel = context.WithCancel(context.Background())
+	m.topics = make(map[string]mqttPubSubSubscription)
```

Comment:
If this is the same change in reelase branch, it should be removed from here to avoid conflict. We merge release branch into master.

Reply:
It's the same. I will change that once it's merged in master

Reply:
The change was merged in the release branch first in this case.